src/components/Rapidoc.js

```diff
@@ -24,28 +23,6 @@ const entityToTagMap = {
   keys: "Keys",
 };
 
-function filterPathsByTag(spec, tags) {
-  const filteredPaths = {};
-  Object.keys(spec.paths).forEach((pathKey) => {
-    const pathItem = spec.paths[pathKey];
-    const methods = Object.keys(pathItem);
-
-    methods.forEach((method) => {
-      if (tags.some((tag) => pathItem[method]?.tags?.includes(tag))) {
-        if (!filteredPaths[pathKey]) {
-          filteredPaths[pathKey] = {};
-        }
-        filteredPaths[pathKey][method] = pathItem[method];
-      }
-    });
-  });
-
-  return {
-    ...spec,
-    paths: filteredPaths,
-  };
-}
-
```

Comment:
Why was this moved?

Reply:
I cleaned this file up. It's quite messy